generate_question.py

```diff
@@ -0,0 +1,26 @@
+import openai
+
+def get_question_and_answer(topic):
+    # this function sends a request to openai api to generate a question about any topic (it takes the topic as a parameter and returns the question and the answer as a dict)
+    prompt = f"Generate a question and answer about the topic: {topic}"
+    try:
+        # Send the prompt to the OpenAI API using the ChatCompletion endpoint
+        response = openai.ChatCompletion.create(
+            model="gpt-3.5-turbo",  # Use "gpt-4" if you have access to it
+            messages=[
+                {"role": "system", "content": "You are a helpful assistant."},
```

Comment:
hi yishai this will not be the final version I pushed this code because I wanted shahen to try to run this part of code on his computer because we had a problem with the api,